morph_client/src/Morph_client.re

```diff
@@ -65,9 +61,13 @@ let read_response = (~notify_finished, response, response_body) => {
       Httpaf.Body.schedule_read(response_body, ~on_read, ~on_eof);
     }
     and on_eof = () => {
-      Lwt.wakeup_later_exn(
+      Lwt.wakeup_later(
```

Comment:
I'm not sure what we want to do here, if we just return the response the user can handle the bad status and do whatever they want with it. In this example I just return the bad request.